apps/bfd-server/bfd-server-war/src/test/java/gov/cms/bfd/server/war/r4/providers/CoverageTransformerV2Test.java

```diff
@@ -759,4 +791,46 @@ static void assertPartDMatches(Beneficiary inBeneficiary, Coverage inCoverage) {
     verifyRelationship();
     verifyPayor();
   }
+
+  static void checkForNoYearlyDate(Beneficiary inBeneficiary, Coverage inCoverage) {
```

Comment:
can this be private? If not and it is shared between tests is there a utility class that it can go in?